float64 probability
float64 xmin
float64 ymin
float64 xmax
float64 ymax
int16 box_num
# string Class
